core/src/edu/unibo/martyadventure/view/character/PlayerCharacterView.java

```diff
@@ -0,0 +1,35 @@
+package edu.unibo.martyadventure.view.character;
+
+import java.util.concurrent.ExecutionException;
+
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.math.Vector2;
+
+import edu.unibo.martyadventure.model.character.PlayerCharacter;
+import edu.unibo.martyadventure.model.character.Shoes;
+
+/**
+ * A player character's base providing basic movement, interaction with given the map
+ * and visual representation.
+ */
+public class PlayerCharacterView extends CharacterView {
+
+    private static PlayerCharacter player = new PlayerCharacter(Shoes.SLOW, "Marty", 100, null);
```

Comment:
Stesso discorso + codice duplicato. Se hai le stesse funzionalità tra tutte le CharacterView, sposta il codice in character view, se è solo tra certi personaggi particolari, fai una classe astratta a parte